app/models/work.rb

```diff
@@ -35,4 +35,11 @@ class Work < ApplicationRecord
       transition deposited: :version_draft
     end
   end
+
+  sig { returns(T.nilable(String)) }
+  def purl
+    return nil unless druid
+
+    File.join(Settings.purl_url, T.must(druid).delete_prefix('druid:'))
```

Comment:
Can you add a test for this?

Reply:
I'm curious about using `File.join` to do a string/URL join.

Reply:
File.join is smart about the trailing slash. (We do the same in Argo: https://github.com/sul-dlss/argo/blob/8c31437630dcce1fdb8f9a9c6fb5f8ff5a8a9a32/app/components/external_links_component.rb#L10)

Reply:
If we were running on Windows, this would be problematic but I'm thinking that is unlikely.

Reply:
Test added.

Reply:
ok, thanks!